src/components/Accounts/Transactions/Transaction/Overview/index.js

```diff
@@ -47,7 +48,15 @@ export default class Overview extends React.Component {
             ) : (
               <img className="transaction--overview--map" src={`https://maps.googleapis.com/maps/api/staticmap?size=640x200&zoom=16&markers=${lat},${long}&scale=1`} />
             )}
-            <img src={logo || require('assets/shopping-bag.svg')} className="transaction--overview--logo rounded" />
+            {logo ?
```

Comment:
Could you amend this code to follow the use brackets I use [here](https://github.com/robcalcroft/mondoweb/blob/master/src/components/Accounts/Transactions/index.js) in the ternary expressions thanks